src/registry_schemas/schemas/correction.json

```diff
@@ -170,6 +170,80 @@
               }
             }
           }
+        },
+        {
+          "if": {
+            "required": ["legalType"],
+            "properties": {
+              "legalType": {
+                "enum": [
+                  "CP"
+                ]
+              }
+            }
+          },
+          "then": {
+            "required": [
+              "contactPoint",
+              "business"
+            ],
+            "properties": {
+              "type": {
+                "type": "string"
+              },
+              "resolutionConfirmed": {
+                "type": "boolean"
+              },
```

Comment:
We've never added these confirmation fields into the schemas.  The FE still saves it for some scenarios but I think we can remove it from the schema definition.

Reply:
removed